url-shortener/backend/src/Backend.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE TypeFamilies #-}
+module Backend where
+
+import Common.Route
+import Control.Concurrent
+import Control.Monad.IO.Class
+import qualified Data.Aeson as A
+import Data.ByteString.Lazy
+import Data.Text (Text)
+import Data.Word
+import Database.Id.Class
+import Data.Pool
+import Database.PostgreSQL.Simple
+import Gargoyle.PostgreSQL.Connect
+import Prelude hiding (id)
+import Obelisk.Backend
+import Obelisk.Route
+import Snap.Core
```

Comment:
Because this is designed to be an example to help people learn, let's add explicit import lists to all of these. It's hard to learn from code when you don't know where the functions are coming from at a glance.

Reply:
Unless they're qualified already...